com.transmissionbt.Transmission.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "com.transmissionbt.Transmission",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "transmission-gtk",
+    "rename-desktop-file": "transmission-gtk.desktop",
+    "rename-icon": "transmission",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "libevent",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://github.com/libevent/libevent.git",
+                    "branch": "release-2.1.8-stable",
+                    "commit": "e7ff4ef2b4fc950a765008c18e74281cdb5e7668"
+                }
+            ]
+        },
+        {
+            "name": "transmission",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://github.com/transmission/transmission.git",
+                    "branch": "2.92",
+                    "commit": "67ef7b888c37fdb85a6d0733c1cc2f0f7c585f66"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "rm m4/glib-gettext.m4"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "appdata-Makefile",
+                    "dest-filename": "Makefile"
+                },
+                {
+                    "type": "file",
```

Comment:
I'd put it into `transmission` module instead and call `post-install` or `build-commands` to install it without a Makefile.